CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ All notable changes to COMMIT will be documented in this file.
 ### Changed
 - core.pyx: The function build_operator checks if the LUT configuration 
             changed before build a new operator
+- trk2dictionary.pyx: Check that the tractogram exists before trying to 
```

Comment:
Put this under "Fixed" to highlight that this modification is a fix

Reply:
OK, done